core/Metrics/Formatter.php

```diff
@@ -71,7 +71,7 @@ public function getPrettyTimeFromSeconds($numberOfSeconds, $displayTimeAsSentenc
             } else {    
                 $time    = sprintf(Piwik::translate('Intl_NDays'), $days) . " " . sprintf("%02s", $hours) . ':' . sprintf("%02s", $minutes) . ':' . sprintf("%02s", $seconds);
             }
-            $centiSeconds = ($numberOfSeconds * 100) % 100;
+            $centiSeconds = intval($numberOfSeconds * 100) % 100;
```

Comment:
note to myself, this avoids `Deprecated: Implicit conversion from float xxxxx to int loses precision`. There was no rounding happening before either https://3v4l.org/OK980 so all good